git/00.setup.md

```diff
@@ -42,8 +42,8 @@ combinations.
 ```bash
 git config --global alias.aa add --all
 git config --global alias.d diff --staged
-git config --global alias.count shortlog -sn
-git config --global alias.l log --pretty=format:%C(yellow)%h%Cblue%d%Creset %s %C(white) %an, %ar%Creset --graph
+git config --global alias.pf push --force-with-lease
+git config --global alias.l log --graph --decorate --oneline
```

Comment:
I tested these alias commands and found that they didn't actually work correctly! I think this may have been part of why your log alias looked weird to me. When I fixed it (see recent commit), it actually looks very much like my log alias except with darker branch names and a little more info on each line. Anyway, see what you think now.